enterprise/javaee.wildfly/src/org/netbeans/modules/javaee/wildfly/ide/WildflyJ2eePlatformFactory.java

```diff
@@ -201,6 +204,9 @@ public Set<String> getSupportedJavaPlatformVersions() {
             if (this.properties.getServerVersion().compareToIgnoreUpdate(WildflyPluginUtils.EAP_7_0) >= 0) {
                 versions.add("17"); // NOI18N
             }
+            if (this.properties.getServerVersion().compareToIgnoreUpdate(WildflyPluginUtils.EAP_8_0) >= 0) {
+                versions.add("26"); // NOI18N
```

Comment:
This makes no sense. This method returns the _java_ platforms supported, there is no JDK 26. Did you mean 21?

Reply:
Removed